share/scripts/inverse/update_ibi_pot.pl

```diff
@@ -56,17 +56,43 @@
 (readin_table($cur_pot_file,@pot_r_cur,@pot_cur,@pot_flags_cur)) || die "$progname: error at readin_table\n";
 
 #should never happen due to resample, but better check
-die "Different grids \n" if (($r_aim[1]-$r_aim[0]-$r_cur[1]+$r_cur[0])>0.0001);
+die "Different grids \n" if (($r_aim[1]-$r_aim[0]-$r_cur[1]+$r_cur[0]) > 0.0001);
 die "Different start potential point \n" if (($r_aim[0]-$r_cur[0]) > 0.0001);
 die "Different end potential point \n" if ( $#r_aim != $#r_cur );
 
 my $outfile="$ARGV[3]";
 my @dpot;
 my @flag;
-my $value=0.0;
 
-#start from the end to make the begining have the last value
-for (my $i=$#r_aim;$i>=0;$i--){
+sub maxindex {
+  my( $aref, $idx_max ) = ( shift, 0 );
+  $aref->[$idx_max] > $aref->[$_] or $idx_max = $_ for 1 .. $#{$aref};
+  return $idx_max;
+}
+my $ndx_max_rdf_cur = maxindex(\@rdf_cur);
+
+#start from the middle and make the outside regions have the last valid value
```

Comment:
Maybe the comments needs an update too ;-)

Not sure it was correct before.

Reply:
done